src/index.js

```diff
@@ -0,0 +1,10 @@
+function say(name)
+{
+  if (!name) return 'Hello world!'
```

Comment:
🙈 Style issue: 
- No space before braces (just as function above)
- Put the `return` on the next line

```ts
if(!name)
   return 'Hello world!'
```